solace/README.md

```diff
@@ -32,10 +33,13 @@ This object MUST NOT contain any properties. Its name is reserved for future use
 
 We need the ability to support several bindings for each operation, see the [Example](#example) section below for details.
 
-Field Name | Type | Description
----|---|---
-`bindingVersion`|String|The current version is 0.3.0
-`destinations`|List of Destination Objects|Destination Objects are described next.
+Field Name | Type                                                                            | Description
+---|---------------------------------------------------------------------------------|---
+`bindingVersion`| String                                                                          |The current version is 0.3.0
+`destinations`| List of Destination Objects                                                     |Destination Objects are described next.
+`timeToLive`  | Integer \| [Schema Object][schemaObject] \| [Reference Object](referenceObject) | Interval in milliseconds or a *Schema Object* containing the definition of the lifetime of the message. 
+`priority`  | Integer \| [Schema Object][schemaObject] \| [Reference Object](referenceObject) | The valid priority value range is 0-255 with 0 as the lowest priority and 255 as the highest or a *Schema Object* containing the definition of the priority.
```

Comment:
>[Schema Object][schemaObject] \| [Reference Object](referenceObject)

This appears to be broken. I think for v3, it's supposed to go here for referenceObject (https://www.asyncapi.com/docs/reference/specification/v3.0.0#referenceObject). Similarly for schemaObject.

Reply:
thanks for pointing out. i missed to links. Added it at the bottm.